p2p/protocol/autonatv2/options.go

```diff
@@ -0,0 +1,48 @@
+package autonatv2
+
+import "time"
+
+// autoNATSettings is used to configure AutoNAT
+type autoNATSettings struct {
+	allowAllAddrs     bool
```

Comment:
I prefer the name you used elsewhere "allowPrivateAddrs"

Reply:
incomplete renaming